frontend/Rhyolite/Frontend/Modal/Base.hs

```diff
@@ -1,3 +1,6 @@
+-- | Modal widgets. The important definition here is `ModalT`, the relative
```

Comment:
Did you perhaps mean 'related' or 'respective'? I guess 'relative' technically makes sense but it sounds odd to me

Reply:
Yes 'related' would be better.

Reply:
Thanks!